packages/sucrase/types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Plugin } from 'rollup';
+type Options = {
```

Comment:
```suggestion
type RollupSucraseOptions = {
```

to follow convention in the repo